### PR TITLE
add part of to 'preorbital bone region' structures

### PR DESCRIPTION
**Pull request by shawntanzk**
Fixes https://github.com/obophenotype/uberon/issues/2936
Note, did not do much research, just following definition as suggested by @aleixpuigb 
Honestly just had a bit of time between finishing my last task and drinks so... lol  
#gogoeditdiff

### CODE REVIEW

**Comment by aleixpuigb** [comment]
Hi @megbalk and @helenamachado, I would like to ask your opinion about this ticket since you added this term in #2416. Given the definition, would it be ok to add the axiom `'part of' some 'preorbital bone region'` to `jugal bone`, `lacrimal bone` and `maxilla`?

**Comment by megbalk** [comment]
@aleixpuigb yep! For maxilla, it is only the part anterior to the orbit. I suppose it could be also added for nasal bones, but only sometimes as it depends on the location of the nasal bones.

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -49873,6 +49875,7 @@ relationship: develops_from UBERON:0010334 {source="EHDAA2"} ! maxillary process
 relationship: in_lateral_side_of UBERON:0011156 {source="FMA-abduced-lr"} ! facial skeleton
 relationship: part_of UBERON:0003113 {notes="facial series", source="ISBN:0073040584"} ! dermatocranium
 relationship: part_of UBERON:0003277 {source="VHOG"} ! skeleton of upper jaw
+relationship: part_of UBERON:7500102 ! preorbital bone region
```
Comment (aleixpuigb) [inline src/ontology/uberon-edit.obo:49878]: I suggest 'overlaps' instead of 'part of'

**Review by aleixpuigb — CHANGES_REQUESTED** [review]
Please, see inline comment.

**Comment by shawntanzk** [comment]
#gogoeditdiff

**Comment by shawntanzk** [comment]
thanks @aleixpuigb & @megbalk - not sure if the other part_of are alright, but I have changed the one you pointed out as suggested. Also realised automated diff didnt run - I dont think there will be any issue, but in case, would be more comfortable letting that run and making sure nothing weird is inferred from these additions

**Review by aleixpuigb — APPROVED** [review]

**Pull request merged**